test/tests/PlayTests.cpp

```diff
@@ -0,0 +1,139 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2018 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "TestData.h"
+
+#include <gtest/gtest.h>
+#include <openrct2/Context.h>
+#include <openrct2/Game.h>
+#include <openrct2/GameState.h>
+#include <openrct2/OpenRCT2.h>
+#include <openrct2/ParkImporter.h>
+#include <openrct2/actions/ParkSetParameterAction.hpp>
+#include <openrct2/actions/RideSetPriceAction.hpp>
+#include <openrct2/object/ObjectManager.h>
+#include <openrct2/peep/Peep.h>
+#include <openrct2/platform/platform.h>
+#include <openrct2/ride/Ride.h>
+#include <openrct2/world/MapAnimation.h>
+#include <openrct2/world/Park.h>
+#include <openrct2/world/Scenery.h>
+#include <openrct2/world/Sprite.h>
+#include <string>
+
+using namespace OpenRCT2;
+
+class PlayTests : public testing::Test
+{
+};
+
+static std::unique_ptr<IContext> localStartGame(const std::string& parkPath)
+{
+    gOpenRCT2Headless = true;
+    gOpenRCT2NoGraphics = true;
+    core_init();
+
+    auto context = CreateContext();
+    if (!context->Initialise())
+        return {};
+
+    auto importer = ParkImporter::CreateS6(context->GetObjectRepository());
+    auto loadResult = importer->LoadSavedGame(parkPath.c_str(), false);
+    context->GetObjectManager().LoadObjects(loadResult.RequiredObjects.data(), loadResult.RequiredObjects.size());
+    importer->Import();
+
+    reset_sprite_spatial_index();
+
+    reset_all_sprite_quadrant_placements();
+    scenery_set_default_placement_configuration();
+    load_palette();
+    map_reorganise_elements();
+    sprite_position_tween_reset();
+    AutoCreateMapAnimations();
+    fix_invalid_vehicle_sprite_sizes();
+
+    gGameSpeed = 1;
+
+    return context;
+}
+
+template<class Fn> static bool updateUntil(GameState& gs, int maxSteps, Fn&& fn)
+{
+    while (maxSteps-- && !fn())
+    {
+        gs.UpdateLogic();
+    }
+    return maxSteps > 0;
+}
+
+TEST_F(PlayTests, SecondGuestInQueueShouldNotRideIfNoFunds)
+{
+    /* This test verifies that a guest, when second in queue, won't be forced to enter
+     * the ride if it has not enough money to pay for it.
+     * To simulate this scenario, two guests (a rich and a poor) are encouraged to enter
+     * the ride queue, and then the price is raised such that the second guest in line
+     * (the poor one) cannot pay. The poor guest should not enter the ride.
+     */
+    std::string initStateFile = TestData::GetParkPath("small_park_with_ferris_wheel.sv6");
```

Comment:
Didn't you forget to include this?

Reply:
This is in the PR isn't it? It's a binary file.